src/storage/txn/actions/prewrite.rs

```diff
@@ -501,12 +501,15 @@ impl<'a> PrewriteMutation<'a> {
 
         match (self.assertion, write) {
             (Assertion::Exist, None) => {
+                info!("assertion failed: exist, previous write record not found";);
```

Comment:
Better to print more information about the current transction, prewriting key/value pair and important flags such as `is_pessimisitic` key etc.